packages/datadog-plugin-aws-sdk/src/services/dynamodb.js

```diff
@@ -133,7 +133,10 @@ class DynamoDb extends BaseAwsSdkPlugin {
     }
     const primaryKeySet = primaryKeyConfig[tableName]
     if (!primaryKeySet || !(primaryKeySet instanceof Set) || primaryKeySet.size === 0 || primaryKeySet.size > 2) {
-      log.warn(`Invalid dynamo primary key config for table ${tableName}: ${JSON.stringify(primaryKeyConfig)}`)
+      log.warn(
+        `span pointers: failed to extract PutItem span pointer: table ${tableName} ` +
+        'not found in primary key names. Please set them through the DD_AWS_SDK_DYNAMODB_TABLE_PRIMARY_KEYS env var.'
```

Comment:
might still be nice to include what we parsed the environment variable, in case there's some parsing or other structural issue.

Reply:
If there's a structural issue, it will get handled and logged in the `getPrimaryKeyConfig()` function.

However, I think we should remove the `primaryKeySet.size > 2` check because there could be a case where the table does exist in the primary key names, but we error anyway when it could have worked fine.

Reply:
(i think the more information we and our customers have while debugging the better, in this case)

Reply:
Fair enough, I'll just update the log to say something more clear like "table {table} not found in primary key names _or the env var was invalid_"